src/headset/oculus_mobile.c

```diff
@@ -0,0 +1,231 @@
+#include "lib/glfw.h"
+#include "headset/headset.h"
+#include "BridgeLovr.h"
+#include "math/quat.h"
+#include "graphics/graphics.h"
+#include "math/mat4.h"
+#include "lib/glad/glad.h"
+
+static void (*renderCallback)(void*);
+static void* renderUserdata;
+
+static float offset;
+
+void lovrOculusMobileDraw(int framebuffer, int width, int height, float *eyeViewMatrix, float *projectionMatrix) {
+  lovrGpuDirtyTexture();
+
+  CanvasFlags flags = {0};
+  Canvas *canvas = lovrCanvasCreateFromHandle(width, height, flags, framebuffer, 0, 0, 1, true);
```

Comment:
Leaving a note about the memory allocation here (can be fixed later post-merge).